templates/patterns/banners/banner_hero.ui_patterns.yml

```diff
@@ -1,6 +1,6 @@
 banner_hero:
-  label: "Hero banner"
-  description: "Hero banner displays a prominent message and related action. It can have an image as background."
+  label: "Hero banner (Deprecated)"
+  description: "Hero banner displays a prominent message and related action. It can have an image as background. The pattern will be deprecated in the next release. Use the Banner pattern instead."
```

Comment:
After this PR merged it is deprecated already, so indicate when this will be removed instead.